### PR TITLE
Init batch 5 mgmt. sdks

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Review by fengzhou-msft — APPROVED** [review]

File: sdk/automation/Azure.ResourceManager.Automation/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  Etag: ETag|etag
+
+request-path-to-parent:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/jobs: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/jobs/{jobName}
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/softwareUpdateConfigurations:  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/softwareUpdateConfigurations/{softwareUpdateConfigurationName}
+override-operation-name:
+  Job_ListByAutomationAccount: GetAll
+operation-positions:
+  Job_ListByAutomationAccount: collection
+  SoftwareUpdateConfigurations_List: collection
```
Comment (fengzhou-msft) [inline sdk/automation/Azure.ResourceManager.Automation/src/autorest.md:60]: Do we know why these GetAll methods are not automatically identified by the generator?

Comment (ArthurMa1978) [reply sdk/automation/Azure.ResourceManager.Automation/src/autorest.md:60]: Good question, need further investigation

**Comment by cataggar** [comment]
Thanks @ArthurMa1978, this is great! I put together this gist for getting started with Azure.ResourceManager.Avs:
https://gist.github.com/cataggar/b707348d18f6a542f8552535a5da02d6

**Pull request merged**